src/ProgressCircle.xml

```diff
@@ -4,16 +4,74 @@
     <properties>
+        <property key="dateSource" type="enumeration" defaultValue="attribute">
+            <caption>Value source</caption>
+            <category>Values</category>
+            <description/>
+            <enumerationValues>
+                <enumerationValue key="attribute">Attribute</enumerationValue>
+                <enumerationValue key="static">Static</enumerationValue>
+            </enumerationValues>
+        </property>
+        <property key="progressAttribute" type="attribute" required="false">
+            <caption>Value attribute</caption>
+            <category>Values</category>
+            <description>When no attribute is provided the 'Value static' will be used to calculate the progress</description>
```

Comment:
`When no attribute is provide, 'Value static' will be used to calculate the progress`

Reply:
Done